### PR TITLE
Help file spacing fix

### PR DESCRIPTION
**Pull request by alexyuwen**
## Purpose and Motivation

Fix spacing.

## Types of changes

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

**Review by capital-G — APPROVED** [review]
Thanks for the PR! The changes look valid to me, let's wait for another pair of eyes to merge it.

**Review by JordanHendersonMusic — APPROVED** [review]

**Comment by JordanHendersonMusic** [comment]
@alexyuwen thank you!

**Pull request merged**
